src/framework/theme/components/user/user.component.ts

```diff
@@ -199,6 +199,10 @@ export class NbUserComponent {
   showInitialsValue: boolean = true;
   isMenuShown: boolean = false;
 
+  get onlyPictureMode(): boolean {
```

Comment:
Considering that this getter is used with negation in the template I would personally prefer to rename it in the reverse version like: `shouldShowUserInfo` or something similar. This way we may remove negations in the template and in the function's code.
What do you think?

Reply:
Also, what about testing?